### PR TITLE
joyshockmapper: Add version 1.6.0

### PR DESCRIPTION
**Pull request by tangmi**

### CODE REVIEW

File: bucket/joyshockmapper.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "version": "1.6.0",
+    "description": "A tool for PC gamers to play games with DualShock 4s, JoyCons, and Pro Controllers. Gyro aiming, flick stick.",
+    "homepage": "https://github.com/JibbSmart/JoyShockMapper",
+    "license": "MIT",
+    "notes": "Directory for auto-load configs can be found at  $persist_dir\\AutoLoad",
+    "persist": ["AutoLoad"],
```
Comment (Ash258) [inline bucket/joyshockmapper.json:7]:
```suggestion
```

File: bucket/joyshockmapper.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "url": "https://github.com/JibbSmart/JoyShockMapper/releases/download/v1.6.0/JSM.zip",
+    "hash": "847ccc4a65cdc38bb1e169782b59b16e9aef78f1d94b0db61bc09864a02fb71b",
+    "extract_dir": "JSM",
+    "shortcuts": [["JoyShockMapper.exe", "JoyShockMapper"]],
```
Comment (Ash258) [inline bucket/joyshockmapper.json:11]:
```suggestion
    "shortcuts": [
        [
            "JoyShockMapper.exe",
            "JoyShockMapper"
        ]
    ],
    "persist": "AutoLoad",
```

File: bucket/joyshockmapper.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+    "version": "1.6.0",
+    "description": "A tool for PC gamers to play games with DualShock 4s, JoyCons, and Pro Controllers. Gyro aiming, flick stick.",
```
Comment (Ash258) [inline bucket/joyshockmapper.json:3]:
```suggestion
    "description": "Converts various controllers types input into keyboard and mouse events, allowing them to be used on PC.",
```

Comment (tangmi) [reply bucket/joyshockmapper.json:3]: JoyShockMapper is unique for what it does with gyroscopic aiming and a novel movement scheme the author calls "flick stick". I'll admit that the description I pulled from the source repo isn't the best, but I'm unsure how to word it to include the differentiating features.

I'm happy to use your suggest if you think it fits best with scoop!

File: bucket/joyshockmapper.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
```
Comment (Ash258) [inline bucket/joyshockmapper.json:3]:
```suggestion
    "description": "Converts various controllers types input into keyboard and mouse events, allowing them to be used on PC.",
```

**Comment by pratikpc** [comment]
> Going forward, the Electronicks fork is the official home of JoyShockMapper. You can find the latest builds and updates there. Just as he has been a huge contributor to JSM while it was mine, I will continue to be involved now that it's his. You can read more about this decision on GyroWiki here.

Could you create a new PR for the same?

**Pull request closed without merging**